versioned_docs/version-8.5/guides/getting-started-java-spring.md

```diff
@@ -1,15 +1,22 @@
 ---
 id: getting-started-java-spring
-title: Getting started as a Java developer using Spring
-sidebar_label: Getting started as a Java developer using Spring
+title: Get started as a Java developer using Spring
+sidebar_label: Get started with Spring
 description: "Use Spring Boot and the Spring Zeebe SDK to interact with your local Self-Managed Camunda 8 installation."
 keywords: [java, spring, spring zeebe, getting started, user guide, tutorial]
 ---
 
+import SmPrereqs from './react-components/sm-prerequisites.md'
+import Install from './react-components/install-plain-java.md'
```

Comment:
These files don't exist for 8.5 and the plain Java install is only available for 8.6.0-alpha2+. The pre-reqs are also different.

Reply:
I botched this when I backported -- will adjust.